### ISSUE TITLE
Nuki Opener - Directly open door with a switch

### ISSUE THREAD

**Issue by nrxofficial** [kvj/hass_nuki_ng#119]
Hi,

i came from homebridge to homeassistant and i miss a function of the homebridge nuki plugin.
I had a Lock in Homebridge/Homekit with the function of opening the door directly, not toggling the ring to open function.
I think i would be useful to add this "Lock" to the HASS plugin, because with the current integration i get the notification, that the lock was locked (even it was locked),

Homebridge example: https://github.com/lukasroegner/homebridge-nuki

**Comment by toooni** [comment]
Hi
I've just fixed the same issue by adding a "script". You can add a script like in the screen capture. After that, you can just add it to HomeKit Bridge with "Scenes" and choose "Switch" on setup.

https://user-images.githubusercontent.com/241080/211269916-b6ff5d4a-3d59-4d1d-8709-294e08fef4e4.mov

**Comment by aebgit** [comment]
> Hi I've just fixed the same issue by adding a "script". You can add a script like in the screen capture. After that, you can just add it to HomeKit Bridge with "Scenes" and choose "Switch" on setup.
> 
>  Screen.Recording.2023-01-09.at.09.47.51.mov

This seems to be for a Nuki Lock, but not the Nuki Opener.

**Comment by toooni** [comment]
@aebgit No. I've done this with the Nuki opener. I do not have a Nuki Lock.
